.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```
@@ -18,10 +21,22 @@
     <MacCatalystProjectFolder Condition=" '$(MacCatalystProjectFolder)' == '' ">MacCatalyst\</MacCatalystProjectFolder>
     <IPhoneResourcePrefix Condition=" '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'maccatalyst' ">$(MacCatalystProjectFolder)Resources</IPhoneResourcePrefix>
     <_SingleProjectMacCatalystExcludes>$(MacCatalystProjectFolder)/**/.*/**</_SingleProjectMacCatalystExcludes>
+
+    <!-- WinUI3 -->
+    <EnableDefaultApplicationDefinition>false</EnableDefaultApplicationDefinition>
+    <EnableDefaultPageItems>false</EnableDefaultPageItems>
+    <WindowsProjectFolder Condition=" '$(WindowsProjectFolder)' == '' ">Windows\</WindowsProjectFolder>
+    <_SingleProjectWindowsExcludes>$(WindowsProjectFolder)/**/.*/**</_SingleProjectWindowsExcludes>
+    <OutputType Condition="'$(OutputType)' == 'Exe' And '$(MSBuildRuntime)' == 'Full' And $([MSBuild]::IsOSPlatform('windows'))">WinExe</OutputType>
   </PropertyGroup>
 
   <!-- Removals -->
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' and '$(SingleProject)' == 'true' ">
+
+    <!-- Remove all Windows (WinUI) XAML Files from the Windows folder -->
+    <EmbeddedResource
+      Remove="$(WindowsProjectFolder)**/*.xaml" />
```

This needs to check if `$(WindowsProjectFolder)` is blank:

```suggestion
    <EmbeddedResource
      Condition="'$(WindowsProjectFolder)' != ''"
      Remove="$(WindowsProjectFolder)**/*.xaml" />
```